### ISSUE TITLE
When a page is opened from outside ...

### ISSUE THREAD

**Issue by johnwcowan** [skyjake/lagrange#625]
... and Lagrange is not running, the new Lagrange window is in front.

But if Lagrange is already running, it opens in a new tab (good) but the window is not brought to the front, so I don't notice that the page has been opened (bad).

Fix: always front the page in this situation.

**Comment by skyjake** [comment]
Which OS does this apply to? SDL's behavior in this area is quite platform-dependent...

**Comment by johnwcowan** [comment]
I've only used Lagrange on Windows, but it seems to me the behavior should be uniform across all platforms: they all have the idea of passing an URL to a browser that can handle it depending on the URL type, so however that happens the main window needs to be surfaced.  But I may not know what I am talking about.

**Comment by skyjake** [comment]
You are right, that is the intended behavior on all platforms. I meant that the native APIs for this have different quirks depending on platform, and while SDL does a good job of hiding any differences, some workarounds may have to be applied manually.
